lib/util/graph.py

```diff
@@ -18,7 +18,7 @@ def get_current_leaf_nodes(list1, list2):
     return list3
 
 
-def get_leaf_nodes(write_to_file=True):
+def get_leaf_nodes(src_file, dest_file):
     """
     This function is used to compute the message-ids of leaf nodes in the thread graph.
 
```

Comment:
@hrishikesh-dahiya The documentation needs update.

Reply:
documentation updated